script/Arcueid/Day 7/07_00_ARC07_3_1_3.txt

```diff
@@ -34,9 +34,9 @@ I knew you would ask that!"
 "I get it, you're thinking ahead after taking down Vlov.
 }
 [sha:b4adf8fe32d48708d818c2add8697e82cf3f9c12]{
--- Page 577, Offset 18020. Glued. Mods: @x.
+-- Page 577, Offset 18020. Mods: @x.
 -- 志貴は食いついてくるわよねー！」
-You're getting really into this, Shiki!"
+ You're getting really into this, Shiki!"
```

Comment:
this and the above change I'm a little lost on. Is this some script thing with the new format?

Reply:
iirc this line wasnt spaced correctly when i playtested it so i had to add a leading space to get it to render correctly